clojure-mode.el

```diff
@@ -192,6 +192,11 @@ Out-of-the box `clojure-mode' understands lein, boot and gradle."
           (and (listp value)
                (cl-every 'stringp value))))
 
+(defcustom clojure-project-root-locating-function nil
+  "Alternative function to locate clojure project root directory, eg. projectile-project-root"
```

Comment:
Why alternative - let's just set it by default to the currently existing function.